src/overlays/actors/ovl_En_Baisen/z_en_baisen.c

```diff
@@ -57,7 +57,7 @@ static ColliderCylinderInit sCylinderInit = {
 
 static u16 sTextIds[] = { 0x2ABD, 0x2ABB, 0x2AD5, 0x2AD6, 0x2AD7, 0x2AD8, 0x2AC6 };
 
-static AnimationHeader* D_80BE8E4C[] = { &object_bai_Anim_0011C0, &object_bai_Anim_0008B4, &object_bai_Anim_008198 };
+static AnimationHeader* sAnimations[] = { &object_bai_Anim_0011C0, &object_bai_Anim_0008B4, &object_bai_Anim_008198 };
 
 static u8 sAnimModes[] = { ANIMMODE_LOOP, ANIMMODE_LOOP };
```

Comment:
Since we use the full `Animations` for the headers, I feel we should also standardize `sAnimModes` to `sAnimationModes`. Thoughts?

Reply:
Sure, that works. There was both `sAnimModes` and `sAnimationModes` before, and I picked the former for this PR, but I can easily switch to the latter.